indico/modules/rb_new/client/js/util.jsx

```diff
@@ -205,3 +205,20 @@ export function boolStateField(name) {
 export function isDateWithinRange(date, dateRange, _toMoment) {
     return date && dateRange.filter((dt) => _toMoment(dt).isSame(date, 'day')).length !== 0;
 }
+
+export function camelizeKeys(obj) {
+    if (_.isArray(obj)) {
+        return obj.map((item) => camelizeKeys(item));
+    }
+
+    const newObject = {};
+    for (const [key, value] of Object.entries(obj)) {
```

Comment:
more-magic version would be like this:

```
const newObject = Object.entries.reduce((obj, [key, value]) => {
    return {...obj, [newKey]: camelizeKeys(value)}
}, {});
```

but regardless, i would support non-object/array values in `camelizeKeys` (and just return the input in that case); that way you don't need the extra check but can always call it